### PR TITLE
Adding SKR Mini screen adaptor

### PR DESCRIPTION
**Pull request by timmit99**
Addition of the SKR Mini screen adaptor which allows a full Mini12864 or similar display to be used on an SKR min E3 V1.2/2.0 which do NOT have a second EXP port. this uses the SPI header pins and routes them to the needed connections and has a working config.

### CODE REVIEW

File: SKR-Mini_Screen_Adaptor/ScreenBreakout.cfg [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#----- Screen Breakout config -----#
+
+# Add [include ScreenBreakout.cfg] into your printer.cfg and uplaod this file into the same directory
```
Comment (KiloQubit) [inline SKR-Mini_Screen_Adaptor/ScreenBreakout.cfg:3]: Replace `uplaod` with `upload`

File: README.md [hunk 2]
```diff
@@ -3,14 +3,16 @@ Cool Hardware designed by the Voron Design group
...
-| Taco Raven          | 3D Printer controller with 8 integrated stepper drivers for use with Klipper        |
-| Voron 0 Display     | Small Form Factor Display with OLED, Rotory Encoder or Buttons and Neopixel LED     |
-| STM32 Mini12864     | A standalone MCU for use with Klipper to drive a FYSETC Mini 12864 Display          |
-| PT100 Stick         | PT100 board in StepStick format                                                     |
-| V0_Micro_Display    | Micro Form Factor Display with OLED, Clickable D-pad and Neopixel LED               |
+| Afterburner_Toolhead_PCB | Breakout PCB for the Afterburner toolhead                                           |
+| Hall_Effect_XY_Endstop   | Replacement for mechanical endstop switches with an adjustable magnetic endstop PCB |
+| Klipper_Expander         | Standalone MCU for use with Klipper with various inputs and outputs                 |
+| Microswitch_Endstop      | PCB for mechanical endstop switches                                                 |
+| Microswitch_Z_Endstop    | PCB for mechancial Z endstop switches                                               |
```
Comment (KiloQubit) [inline README.md:10]: Replace `mechancial` with `mechanical`

File: README.md [hunk 3]
```diff
@@ -3,14 +3,16 @@ Cool Hardware designed by the Voron Design group
...
+| Nano_Display             | Adapts an Arduino Nano to the RepRap standard 2 2x5pin IDC Connectors               |
+| PT100Stick               | PT100 board in StepStick format                                                     |
+| SKR-Mini_Screen_Adaptor  | PCB to connect a 2 2x5 IDC cable screen to an SKR Mini E3 V1.2/2.0                  |
+| STM32_Mini12864          | A standalone MCU for use with Klipper to drive a FYSETC Mini 12864 Display          |
+| Taco_Raven               | 3D Printer controller with 8 integrated stepper drivers for use with Klipper        |
+| V0_Display               | Small Form Factor Display with OLED, Rotory Encoder or Buttons and Neopixel LED     |
```
Comment (KiloQubit) [inline README.md:16]: Replace `Rotory` with `Rotary`

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Please resolve README conflicts.

Remove space from `Schematic Layout.png` filename

File: SKR-Mini_Screen_Adaptor/ScreenBreakout.cfg (hunk 1 above)
Comment (timmit99) [reply SKR-Mini_Screen_Adaptor/ScreenBreakout.cfg]: Fixed!

File: README.md (hunk 2 above)
Comment (timmit99) [reply README.md]: Fixed!

File: README.md (hunk 3 above)
Comment (timmit99) [reply README.md]: Fixed!

**Review by timmit99 — COMMENTED** [review]
Changes are complete!

**Review by KiloQubit — APPROVED** [review]

**Pull request merged**